app/templates/games/list.hbs

```diff
@@ -0,0 +1,17 @@
+{{!-- Only allow game creation if you are not anonymous. --}}
+{{#unless session.user.isAnonymous}}
```

Comment:
also, we can show this button but have it disabled to show off functionality that you'll get if you sign in